samwise.js

```diff
@@ -1,14 +1,14 @@
-const reversedString = string.split("").reverse().join("");
```

Comment:
SCRIPT-This new feature lacks unit tests. Could you add some to ensure everything works correctly?